lms/templates/learner_dashboard/course_enroll.underscore

```diff
@@ -1,8 +1,5 @@
 <% if (is_enrolled && collectionCourseStatus === 'completed') { %>
-    <div class="final-grade">
-        <div class="grade-header"><%- gettext('Final Grade') %><span class="sr"><%- StringUtils.interpolate(gettext('for {courseName}'), {courseName: title}) %></span></div>
-        <div class="grade-display"><%- grade %>%</div>
-    </div>
+    <% // do nothing %>
```

Comment:
I understand this is more convenient, but can we rewrite the conditions so this doesn't need to be a separate case from the "else"?